File: JavaScript/min&maxArray.js
function max(array) {
    let highestNum = array[0]
    for (let i = 0; i < array.length; i++) {
        let currentNum = array[i] //
        if (currentNum > highestNum) {
            highestNum = currentNum
        }
    }

    return highestNum;
}
console.log(max([100, 42, 300, 425, 500]))

function min(array) {
    let lowestNum = array[0]
    for (let i = 0; i < array.length; i++) {
        let currentNum = array[i] //
        if (currentNum < lowestNum) {
            lowestNum = currentNum
        }
    }

    return lowestNum;
}
console.log(min([100, 42, 300, 425, 500]))

File: Python/min&maxArray.py
def max(arr):
    highestNum = arr[0]
    for x in arr:
        currentNum = x
        if currentNum > highestNum:
            highestNum = currentNum
    return highestNum

print(max([23, 124, 11, 140]))

def min(arr):
    lowestNum = arr[0]
    for x in arr:
        currentNum = x
        if currentNum < lowestNum:
            lowestNum = currentNum
    return lowestNum

print(min([23, 124, 11, 140]))